api/workers/execution-queue.worker.js

```diff
@@ -3,10 +3,19 @@ const executionManager = require("../execution-manager");
 
 
 async function processRequest(executionRequest) {
-    const executionResult = await executionManager.execute(executionRequest);
-    await amqplib.sendToQueue(process.env.AMQP_RESULT_QUEUE, amqplib.VHOST_RESULTS, Buffer.from(JSON.stringify(executionResult)));
+  const executionResult = await executionManager.execute(executionRequest);
+  await amqplib.sendToQueue(
+    process.env.AMQP_RESULT_QUEUE,
+    amqplib.VHOST_RESULTS,
+    Buffer.from(JSON.stringify(executionResult))
+  );
 }
 
 module.exports = async () => {
-    return amqplib.consumeQueue(`${process.env.AGENT_KEY}-queue`, amqplib.VHOST_ACTIONS, processRequest);
-}
+  // TODO Uncomment when Big Bird will manage queues creating
```

Comment:
I think it may be incorrect. We will still receive logs about AMQP on the setup and this is commented? I thought more of commenting out the setup for the consumer in the bootstrap.